src/public/meeting.ts

```diff
@@ -325,4 +349,22 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel);
     sendMessageToParent('meeting.getAppContentStageSharingState', callback);
   }
+
+  /**
+   * Registers a handler for changes to speaking array.
+   * @param handler The handler to invoke when the array of participants speaking changes. A value
+   * of 0 (no participants speaking) or 1 (one or more participants are speaking) is provided.
+   * @param includeParticipants is a flag to opt-in to receive participant ids of who is speaking.
+   * Default value is false.
```

Comment:
This comment doesn't make much sense either. It says you get `0` or `1`, which isn't what the types below suggest. And the second comment says `to receive participant ids` - receive where, if the callback gets a `0` or `1`?

Reply:
That's true, this should instead align with the types below. The callback should get the type `ISpeakingState ` -- with a boolean for `isSpeakingDetected` and if requested by the developer (using the `includeParticipants` flag), an array of `IParticipantSpeakingState `.

Reply:
Hi @MarcinJuraszek I updated the comment, please feel free to let me know if you have any thoughts on the change.